dynatrace/notebook.tf

```diff
@@ -58,11 +58,82 @@ locals {
           visualization = "table"
         }
       },
+  {
```

Comment:
Should we add a description that this step is only executed if the response time path is followed? And one more question: how is the differentiaton actually made? Is there a new annotation? I probably overlooked it.